package.json

```diff
@@ -76,7 +76,8 @@
     "@ciscospark/spark-core": "1.40.0",
     "@ciscospark/storage-adapter-local-forage": "1.40.0",
     "@ciscospark/storage-adapter-local-storage": "1.40.0",
-    "@collab-ui/react": "^8.0.2",
+    "@collab-ui/core": "^10.25.0",
```

Comment:
Doesn't the react library already pull in core? Or did we need to use a specific version?

Reply:
specific version needed :)